networking/ipv6.md

```diff
@@ -152,66 +141,31 @@ then the following changes will ensure IPv6 operation.
--   DHCP enabled
--   IPv6 address mode set to DHCPv6 stateful.
+#### Configure the Kubernetes control plane to operate over IPv6
 
-We suggest initially configuring both IPv4 and IPv6 subnets in each
-network. This allows handling VM images that support only IPv4 alongside
-those that support both IPv4 and IPv6, and allows a VM to be accessed
-over IPv4 in case this is needed to troubleshoot any issues with its
-IPv6 configuration.
+(Optional). Depending on your networking set up, you may want to configure the Kubernetes control plane to operate over IPv6 for dual stack or IPv6 only.
+ 
+To configure Kubernetes components, enable IPv6 using the following flags.  
 
-In principle, though, we are not aware of any problems with configuring
-and using IPv6-only networks in OpenStack.
+| Component                   | **Flag**                                      | **Value/Content**                                            |
+| --------------------------- | --------------------------------------------- | ------------------------------------------------------------ |
+| **kube-apiserver**          | `--bind-address` or `--insecure-bind-address` | Set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
+|                             | `--advertise-address`                         | Set to the IPv6 address that nodes should use to access the `kube-apiserver`. |
+|                             | `--service-cluster-ip-range`                  | Set to an IPv6 CIDR that will be used for the Service IPs. The DNS service address must be in this range. |
+| **kube-controller-manager** | `--master`                                    | Set with the IPv6 address where the `kube-apiserver` can be accessed. |
+|                             | `--cluster-cidr`                              | Set to match the {{site.prodname}} IPv6 IPPool.              |
+| **kube-scheduler**          | `--master`                                    | Set with the IPv6 address where the `kube-apiserver` can be accessed. |
+| **kubelet**                 | `--address`                                   | Set to the appropriate IPv6 address or `::` for all IPv6 addresses. |
+|                             | `--cluster-dns`                               | Set to the IPv6 address that will be used for the service DNS; this must be in the range used for `--service-cluster-ip-range`. |
+|                             | `--node-ip`                                   | Set to the IPv6 address of the node.                         |
+| **kube-proxy**              | `--bind-address`                              | Set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
+|                             | `--master`                                    | Set with the IPv6 address where the `kube-apiserver` can be accessed. |
+|                             | `--cluster-cidr`                              | Set to match the {{site.prodname}} IPv6 IPPool.  
```

Comment:
Delete row, because this is about workloads, not Kubernetes own control plane.

Reply:
fix.